src/CSharp/EasyMicroservices.Serialization.Tests/Providers/TextSerialization/SystemTextXmlSerializationTest.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using EasyMicroservices.Serialization.System.Text.Xml.Providers;
+using EasyMicroservices.Serialization.Tests.Providers.Models;
+using Xunit;
+
+namespace EasyMicroservices.Serialization.Tests.Providers.TextSerialization
+{
+    public class SystemTextXmlSerializationTest : BaseTextSerializationTest
+    {
+        public SystemTextXmlSerializationTest() : base(new SystemTextXmlProvider())
+        {
+        }
+
+        [InlineData("Mahdi", 30, Gender.Male, "<?xml version=\"1.0\" encoding=\"utf-16\"?>\r\n<ClassToSerialize xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xmlns:xsd=\"http://www.w3.org/2001/XMLSchema\">\r\n  <Name>Mahdi</Name>\r\n  <Age>30</Age>\r\n  <Gender>Male</Gender>\r\n</ClassToSerialize>")]
```

Comment:
Please change the "\r\n" to "Environment.NewLine".
Check the JSON serialization test.
it will not work on Linux and mac.
Thanks.